databricks_cli/clusters/cli.py

```diff
@@ -38,54 +37,58 @@
               help='File containing JSON request to POST to /api/2.0/clusters/create.')
 @click.option('--json', default=None, type=JsonClickType(),
               help=JsonClickType.help('/api/2.0/clusters/create'))
+@profile_option
 @require_config
 @eat_exceptions
-def create_cli(json_file, json):
+def create_cli(api_client, json_file, json):
     """
     Creates a Databricks cluster.
 
     The specification for the request json can be found at
     https://docs.databricks.com/api/latest/clusters.html#create
     """
-    json_cli_base(json_file, json, create_cluster)
+    json_cli_base(json_file, json, lambda json: ClusterApi(api_client).create_cluster(json))
```

Comment:
Could `ClusterApi(api_client)` be saved by a `@provide_cluster_api` decorator?

Reply:
IMO, it's a small optimization that doesn't seem worth the complexity of adding another decorator.